cohortextractor/tpp_backend.py

```diff
@@ -1789,6 +1789,17 @@ def patients_with_test_result_in_sgss(
                 "  test_result = 'positive'"
             )
 
+        if (
+            returning == "number_of_matches_in_period"
+            and restrict_to_earliest_specimen_date is not False
```

Comment:
I can see why you've done it this way here, but in general I'd write `if x` rather than `if x is not False` as the latter will make most Python readers have to think a bit harder.